.github/workflows/pkgdown.yaml

```diff
@@ -0,0 +1,113 @@
+          $outputPath = "${{ runner.workspace }}/setup.zip"
+          Invoke-WebRequest -Uri $url -OutFile $outputPath
+        shell: powershell
+
+      - name: Unzip PK-sim build archive
+        run: |
+          $zipPath = "${{ runner.workspace }}/setup.zip"
+          $unzipPath = "${{ runner.workspace }}/unzipped"
+          Expand-Archive -Path $zipPath -DestinationPath $unzipPath
+        shell: powershell
+
+      - name: Install PK-sim MSI
+        run: |
+          $msiPath = Get-ChildItem -Path "${{ runner.workspace }}/unzipped" -Filter "*.msi" | Select-Object -First 1
+          if ($msiPath) {
+            Start-Process msiexec -ArgumentList "/i", $msiPath.FullName, "/quiet", "/qn", "/norestart" -Wait
+          } else {
+            Write-Output "No MSI file found in the unzipped directory."
+            exit 1
+          }
+        shell: powershell
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Download rClr
+        run: |
+          RCLR_URL <- "https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v${{ env.RCLR_VERSION }}/rClr_${{ env.RCLR_VERSION }}.zip"
+          download.file(RCLR_URL, "rClr_${{ env.RCLR_VERSION }}.zip")
+        shell: Rscript {0}
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: |
+            rClr=local::./rClr_${{ env.RCLR_VERSION }}.zip
+            any::pkgdown
+            local::.
+            ospsuite.utils=github::Open-Systems-Pharmacology/OSPSuite.RUtils
+            tlf=github::Open-Systems-Pharmacology/TLF-Library
+            ospsuite=github::Open-Systems-Pharmacology/OSPSuite-R
+            ospsuite.parameteridentification=github::Open-Systems-Pharmacology/OSPSuite.ParameterIdentification
+          needs: website
+
+      - name: Build site
+        run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
+        shell: Rscript {0}
+
+      - name: Upload Artifacts 🔺 # The project is then uploaded as an artifact named 'site'.
+        uses: actions/upload-artifact@v1
+        with:
+          name: pkgdown
+          path: docs
+
+
+  deploy:
+    concurrency:
+      group: pkgdown-${{ github.event_name != 'pull_request' || github.run_id }}
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+    needs: [pkgdown] # The second job must depend on the first one to complete before running and uses ubuntu-latest instead of windows.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v3
+
+      - name: Download Artifacts 🔻 # The built project is downloaded into the 'site' folder.
+        uses: actions/download-artifact@v1
+        with:
+          name: pkgdown
+
+      - name: Deploy to GitHub pages 🚀
+        if: github.event_name != 'pull_request'
+        uses: JamesIves/github-pages-deploy-action@v4.4.1
```

Comment:
Does it mean we are relying on someone else's code (that can break down at some point) to deploy the website?

Reply:
Yes, these are like github action packages. They are widely used so I don't think they can break things this easily, especially since a specific version is used, 4.4.1 in this case that will not change in the future. Even if they would break for any reason, the community using them is so important that it would be fixed quiet quickly I think.